File: features/merchant/shop_common/src/main/java/com/tokopedia/shop/common/domain/interactor/model/favoriteshop/DataFollowShop.java

package com.tokopedia.shop.common.domain.interactor.model.favoriteshop;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DataFollowShop {

    @SerializedName("followShop")
    @Expose
    private FollowShop followShop;

    public FollowShop getFollowShop() {
        return followShop;
    }

    public void setFollowShop(FollowShop followShop) {
        this.followShop = followShop;
    }

}

File: features/merchant/shop_common/src/main/java/com/tokopedia/shop/common/domain/interactor/model/favoriteshop/DataFollowShop.kt
package com.tokopedia.shop.common.domain.interactor.model.favoriteshop

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DataFollowShop {
    @SerializedName("followShop")
    @Expose
    var followShop: FollowShop? = null
}